app/Models/Location.php

```diff
@@ -17,6 +18,16 @@
 
 class Location extends SnipeModel
 {
+    function __construct() {
```

Comment:
I get the intent but I'd like to see if there is another way to approach this. Can this be handled in the `booted` method?